.envrc

```diff
@@ -47,61 +59,34 @@ require() {
     command -v "$1" >/dev/null 2>&1
 }
 
-warn() {
-    cat <<EOF
-${yellow}${bold}direnv: ${@}${reset}
-EOF
+debug() {
+    if [ "${SENTRY_DIRENV_DEBUG}" ]; then
+        echo -e "${@}"
+    fi
 }
 
 info() {
-    cat <<EOF
-${bold}direnv: ${@}${reset}
-EOF
-    # info() calls happen multiple times in this script. The last info statement
-    # is the one that will be used to group Sentry issues
-    error_message=("${@}")
+    echo -e "${bold}${*}${reset}"
```

Comment:
I'm using now `echo -e` instead of `cat` since it permits including `\n` in order to create blank spaces between messages if we wish to.